docs/application/dotnet/guides/nui/window.md

```diff
@@ -313,6 +313,33 @@ child.Unparent();
 > If parent's window stack is changed by calling `Raise()` or `Lower()`, the child window is located on top of the parent again.
 >
 
+### Window Type
+
+NUI Window has serveral window types.
+The detail descrition is written in NUI Constants. See [WindowType] (application/dotnet/api/TizenFX/latest/api/Tizen.NUI.WindowType.html).
+In addition, we can set window type in NUIApplicatiion's constructor.
+For more information, see [NUIApplication](/application/dotnet/api/TizenFX/latest/api/Tizen.NUI.NUIApplication.html) API.
+Especially, it is only way to create an IME window.
```

Comment:
When you create an application, to create an IME window you must set the window type as IME.
```suggestion
Especially, it is the only way to create an IME window.
```